www/templates/main-goals.html

```diff
@@ -33,23 +33,25 @@
           </div>
         </a>
       </div>
+      <div>
       <div class="row">
-        <div class="col">
-            <img src="img/ic_header_gold.png" style="height:15px; width:15px;">
-            <img src="img/ic_header_silver.png" style="height:15px; width:15px;">
-            <img src="img/ic_header_gem.png" style="height:15px; width:15px;">
+        <div class="col" style="margin:auto; margin-top:5px; margin-left:2.5%">
+            <img src="img/ic_header_gold.png" style="height:15px; width:15px;float: left;"><p style="font-size: 10px; float: left; margin: 0 10% 0 2%;">0</p>
+            <img src="img/ic_header_silver.png" style="height:15px; width:15px;float: left;"><p style="font-size: 10px; float: left; margin: 0 10% 0 2%;">0</p>
+            <img src="img/ic_header_gem.png" style="height:15px; width:15px;float: left;"><p style="font-size: 10px; float: left; margin: 0 0 0 2%;">0</p>
         </div>
-        <div class="col col-50" style="margin-top:-5px;">
-          <img src="img/ic_header_healer.png" style="height:40px; width:40px;" class="center">
-          <h2 style="font-size:12px; text-align:center; margin:0px">{{user.data.profile.name}}</h2>
-          <p style="font-size:10px; text-align:center;">{{user.data.stats.class}}</p>
+        <div class="col col-40" style="margin-top:-5px;">
+          <img src="img/ic_header_healer.png" style="height:30px; width:30px;" class="center">
```

Comment:
I'm going to ask the same question as I always ask Kris. Is it possible to make this be relative, or to put it into css?